andela/hc-aces-kla
# Ft department view checks #158174596

## Status
**READY**

## Migrations
YES

## Description
Different departments within a team should be able to categorise jobs within their domains to a separate view. I have integrated filtering of jobs by departments

## Related PRs
None


## Todos
- [x] views
- [x] checks UI


## Deploy Notes
On deployment run migrations ```./manage.py migrate```

Run background processes  ```./manage.py ensuretriggers``` and ```./manage.py sendalerts```

## Steps to Test or Reproduce

```sh
To assign department to check:
- navigate to the name of a check
- On the displayed form add a department(s)
- save the entry

to view checks by department:
- navigate to each department link displayed below 
  My Checks 
```

## Impacted Areas in Application

* /checks/ page
* checks model
* my_ckecks method in hc/front/views.py

## Snapshots 
![image](https://user-images.githubusercontent.com/30747298/42643462-da7a10f6-8601-11e8-986d-58b909aea6ab.png)

![image](https://user-images.githubusercontent.com/30747298/42643561-18f971fa-8602-11e8-9a6b-0459d33877a8.png)
